### PR TITLE
Added steamname resolver

### PR DESCRIPTION
**Pull request by gpotter2**
Hey !

Just wanted to help cause the "steamid.io" thing sounded complicated for some people i know, so i added a system to resolve directly the steamnames into the 64id. I corrected a few bugs that also appeared as I was trying to do that...

* Corrected a bug with the max nodes selector (got stuck on the max/min)
* Corrected a bug updating continously the 64TargetID (because CellEnter is called eachtime the grid is updated, so i just replaced it by CellClick)
* Added a "more" button to access the steamname resolver (what i was talking about)
* Auto magnified the code (caps + spaces) (that's the auto function... it didn't break anything, i checked)

Note: to resolve the steamname, i'm just parsing the id at the top of the (official) user page:
http://steamcommunity.com/id/<username>/?xml=1

BTW: this project is just awesome !

### CODE REVIEW

File: DaS-PC-MPChan/DSCM.vb [hunk 1]
```diff
@@ -371,10 +371,9 @@ Public Class DSCM
                 nmbMaxNodes.Enabled = True
                 nmbMaxNodes.BackColor = New Color()
             Else
-                nmbMaxNodes.Enabled = False
```
Comment (Chronial) [inline DaS-PC-MPChan/DSCM.vb]: What is your reasoning for this change? This line was here for a reason ^^.

Comment (Chronial) [inline DaS-PC-MPChan/DSCM.vb]: Ah, I see – this is supposed to fix #66. This is not the correct fix for that issue. The actual bug is in line 369. I will try to take a look at this tomorrow.

**Comment by Chronial** [comment]
Wait – this doesn't work at all. It needs needs the user's steamid as input, not the steamname.

Or am I missing something?

**Comment by Wulf2k** [comment]
People can copy and paste the entire profile URL if they'd like, but I don't want to assume anything in there is a profile name.

**Comment by Chronial** [comment]
@gpotter2 I added support for arbitrary profile URLs in faa3a3d919. Does that satisfy your requirements and can this PR be closed?

**Comment by gpotter2** [comment]
@Chronial great !

**Pull request closed without merging**